### ISSUE TITLE
Invalid sge job name as defined in 'sge_types'.

### ISSUE THREAD

**Issue by ezorita** [nextflow-io/nextflow#299]
If `tag` is used with invalid characters, the process is denied when submitted to the SGE cluster. e.g.

Contents of .command.run:
```
#!/bin/bash
#$ -wd /nextflow/work/15/e540e37e06acc9f56c96aa0c8b4366
#$ -N nf-downloadReferenceGenome_(http_//hgdownload.cse.ucsc.edu/goldenPath/hg19/bigZips/chromFa.tar.gz)
```
results in:
```
ERROR ~ Error executing process > 'downloadReferenceGenome (http://hgdownload.cse.ucsc.edu/goldenPath/hg19/bigZips/chromFa.tar.gz)'
Caused by:
  Failed to submit job to grid scheduler for execution
Command executed:
  qsub -terse .command.run
Command exit status:
  2
Command output:
  Unable to read script file because of error: ERROR! argument to -N option must not contain / 
```

As defined in `sge_types`: _The name may be any arbitrary alphanumeric ASCII string, but may not contain  "\n", "\t", "\r", "/", ":", "@", "\", "*",  or "?"._

The job names should be automatically updated to meet these requirements.

nextflow version 0.23.4.4170

**Comment by pditommaso** [comment]
Thanks for reporting this. You can workaround the issue by removing the `/` in the file path.

**Comment by pditommaso** [comment]
Included in version 0.24.0
